boards/arm/nrf52840_pca10056/nrf52840_pca10056.dts

```diff
@@ -25,6 +25,10 @@
 	status = "ok";
 };
 
+&radio0 {
+	compatible = "nordic,nrf52840-radio";
```

Comment:
You don't need the compatible property here if it does not override something. You need `status = "ok";` to enable the node.